src/controllers/v1/did/DidController.ts

```diff
@@ -28,18 +29,27 @@ export class DidController extends Controller {
   @Response<BadRequest['message']>(400)
   @Response<HttpResponse>(500)
   public async getCredentials(
+    @Request() req: express.Request,
     @Query('createdLocally') createdLocally: boolean,
     @Query('method') method?: string
   ): Promise<DidResolutionResultProps[]> {
     if (!createdLocally) {
+      req.log.warn('can list only local DIDs %s', method)
       throw new BadRequest('Can only list DIDs created locally')
     }
 
     const didResult = await this.agent.dids.getCreatedDids({
       method,
     })
 
-    return await Promise.all(didResult.map(({ did }) => this.agent.dids.resolve(did)))
+    req.log.info('attempting to resolve DIDs found %j', didResult)
+
+    return await Promise.all(
+      didResult.map(({ did }) => {
+        req.log.info('resolving %s did', did)
```

Comment:
I think this sjhould be at a debug level at highest. There could be many dids in this list

Reply:
set to `debug`